Makefile

```diff
@@ -549,7 +549,7 @@ enter:
 
 # grpc generates GRPC stubs from service definitions
 .PHONY: grpc
-grpc:
+grpc: update-vendor-api
 	make -C build.assets grpc
```

Comment:
```suggestion
	$(MAKE) -C build.assets grpc
```

Reply:
What's the reason for changing this?

Reply:
It's [recommended for recursive invocations](https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html) and also allows you to override the make command if needed.